### ISSUE TITLE
Training or Testing？

### ISSUE THREAD

**Issue by kaigelee** [amro-kamal/ObjectPose#1]
Thanks for your great work！
I would like to ask if the dataset you provided is just for testing the robustness of the pretrained network?
And can you provide detailed details on generating the ObjectPose dataset?

**Comment by stedeny** [comment]
Yes, it's only data for testing, the dataset is relatively small and could not be used for training these very large models.
